lib/measures/tbd/resources/psi.rb

```diff
@@ -588,7 +592,7 @@ def derate(os_model, os_surface, id, surface, c, index, type, r)
   return m
 end
 
-def processTBD(os_model, io_path, schema_path, psi_set)
+def processTBD(os_model, psi_set, io_path = nil, schema_path = nil)
```

Comment:
@rd2 I propose changing the order of these inputs so the optional ones can be at the end.